.github/workflows/web.yml

```
@@ -180,7 +180,7 @@ jobs:
     with:
       name: component
       module_name: centreon
-      specs_path: www/front_src/src/**
+      specs_path: www/front_src/src/**/
```

dictory inputs should not be suffixed by `/`
```suggestion
      specs_path: www/front_src/src/**
```